src/overlays/actors/ovl_Dm_Stk/z_dm_stk.c

```diff
@@ -26,10 +26,10 @@ void DmStk_ClockTower_DoNothing(DmStk* this, GlobalContext* globalCtx);
 void DmStk_DoNothing(DmStk* this, GlobalContext* globalCtx);
 void DmStk_WaitForTelescope(DmStk* this, GlobalContext* globalCtx);
 void DmStk_StartTelescopeCutscene(DmStk* this, GlobalContext* globalCtx);
-void DmStk_ClockTower_StartFirstTimeIntroCutscene(DmStk* this, GlobalContext* globalCtx);
-void DmStk_ClockTower_WaitForFirstTimeIntroCutsceneToEnd(DmStk* this, GlobalContext* globalCtx);
-void DmStk_ClockTower_StartSubsequentTimesIntroCutscene(DmStk* this, GlobalContext* globalCtx);
-void DmStk_ClockTower_WaitForSubsequentTimesIntroCutsceneToEnd(DmStk* this, GlobalContext* globalCtx);
+void DmStk_ClockTower_StartFirstVersionIntroCutscene(DmStk* this, GlobalContext* globalCtx);
+void DmStk_ClockTower_WaitForFirstVersionIntroCutsceneToEnd(DmStk* this, GlobalContext* globalCtx);
+void DmStk_ClockTower_StartSecondVersionIntroCutscene(DmStk* this, GlobalContext* globalCtx);
+void DmStk_ClockTower_WaitForSecondVersionIntroCutsceneToEnd(DmStk* this, GlobalContext* globalCtx);
```

Comment:
Oh, I was thinking of on the end, so StartIntroCutsceneFirstVersion (or StartIntroCsVersion1, even)

Reply:
Ah, I see. Let me fix that really quick.